service/controller/prometheus_resource_set.go

```diff
@@ -1,8 +1,7 @@
-package v1
+package controller
```

Comment:
I moved this file to a proper place. No interesting changes there besides wiring `reloadResource`.